website/static/js/tests/fangorn.test.js

```diff
@@ -209,21 +209,6 @@ describe('fangorn', () => {
                 item.data.provider = 'github';
                 assert.equal(Fangorn.isInvalidDropItem(folder, item, false, true), true);
             });
-
-            it('cannot be dropped if item provider is figshare and private', () => {
```

Comment:
These test are no longer appropriate, but new tests have been added to `getCopyMode` and `allowedToMove` to update new behavior.